synapse/models/inet.py

```diff
@@ -692,45 +853,45 @@ def getBaseModels():
 
                 )),
 
-                ('inet:netuser', {'ptype': 'inet:netuser'}, [
+                ('inet:web:acct', {'ptype': 'inet:web:acct'}, [
                     ('site', {'ptype': 'inet:fqdn', 'ro': 1}),
                     ('user', {'ptype': 'inet:user', 'ro': 1}),
 
                     ('dob', {'ptype': 'time'}),
 
-                    # ('bio:bt',{'ptype':'wtf','doc':'The netusers self documented blood type'}),
+                    # ('bio:bt',{'ptype':'wtf','doc':'The web account's self documented blood type'}),
 
                     ('url', {'ptype': 'inet:url'}),
                     ('webpage', {'ptype': 'inet:url'}),
                     ('avatar', {'ptype': 'file:bytes'}),
 
-                    ('tagline', {'ptype': 'str:txt', 'doc': 'A netuser status/tag line text'}),
-                    ('occupation', {'ptype': 'str:txt', 'doc': 'A netuser self declared occupation'}),
+                    ('tagline', {'ptype': 'str:txt', 'doc': 'A web account status/tag line text'}),
+                    ('loc', {'ptype': 'str:lwr', 'doc': 'The web account self declared location'}),
+                    ('occupation', {'ptype': 'str:txt', 'doc': 'A web account self declared occupation'}),
```

Comment:
Do we want to make this `str:lwr` ?  That way "Software Engineer" and "software engineer" fold together.

Reply:
I agree this should probably be changed. I'll add tracking for this but not change it as part of this PR.